src/styles/components/ec-card/ec-card.css

```diff
@@ -0,0 +1,38 @@
+:root {
+  --ec-card-interactive-transition-duration: 0.4s;
+  --ec-card-interactive-transition-timing-function: ease;
+}
+
+.ec-card {
+  @apply tw-rounded tw-p-8 tw-bg-light tw-shadow-level-1;
```

Comment:
I'm not very big fan of having one singleline apply call. It would be better to split them into categories, like box-model, position, flex, colors, etc.

```
@apply tw-rounded tw-shadow-level-1;
@apply tw-p-8;
@apply tw-bg-light;
```

I think it's more readable and more organized.